packages/diagram-common/src/model/elements/root.ts

```diff
@@ -19,9 +19,9 @@ export interface Root extends Element {
      */
     fonts: FontFamilyConfig[];
     /**
-     * The bounds of the whole diagram
+     * The bounds of the whole diagram, as defined by hylimo internally, not by sprotty
```

Comment:
I would prefer not to mention sprotty here, as the idea was to have this module completely sprotty-agnostic